webui/react/src/components/Breadcrumb.stories.tsx

```diff
@@ -31,10 +31,10 @@ export const TrialDetail = (): React.ReactNode => (
       </Space>
     </Breadcrumb.Item>
     <Breadcrumb.Item>
-      <Link path="/ui/experiments/3"> Experiment 3</Link>
+      <Link path="/det/experiments/3"> Experiment 3</Link>
     </Breadcrumb.Item>
     <Breadcrumb.Item>
-      <Link path="/ui/experiments/3/trials/34">Trial 34</Link>
+      <Link path="/ui/trials/34">Trial 34</Link>
```

Comment:
for our storybook I think we could go ahead with using the react one :thinking: . either way works